hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java

```diff
@@ -584,7 +584,9 @@ private static void deleteKeysFromDelKeyTableInSnapshotScope(
 
   // Get OmSnapshot if the keyname has ".snapshot" key indicator
   public IOmMetadataReader checkForSnapshot(String volumeName,
-                                            String bucketName, String keyname)
+                                            String bucketName,
+                                            String keyname,
+                                            boolean override)
```

Comment:
more informative param name
```suggestion
                                            boolean skipActiveCheck)
```